README.md

```diff
@@ -267,6 +267,9 @@ export default DS.JSONAPIAdapter.extend(DataAdapterMixin, {
 });
 ```
 
+**If you set the authorizer to authorizer:application** you need to create an `authorizers/application.js` inside your application (app/authorizers/applications.js). See the following section on implementing a custom authorizer.
+
```

Comment:
I see what you're saying and I agree, but the issue I had (and It could be simply because I don't have that much experience with Ember in general) was that I read here in one of the issues (or possibly seeing the dummy app) that you needed to set `authorizer: 'authorizer:application'`. 

But then I went and did that and of course then I got an error of undefined (issue #670). So I thought that for people like me this reference would be a good example to not trip on this. Because effectively when you set the authorizer to something it expects to find a file somewhere. That was the reasoning for adding that line.